core/store/src/flat/delta.rs

```diff
@@ -16,9 +16,20 @@ pub struct FlatStateDelta {
     pub changes: FlatStateChanges,
 }
 
+/// Information about a block.
+#[derive(BorshSerialize, BorshDeserialize, Debug, Clone, Copy, serde::Serialize)]
+pub struct PrevBlockWithChanges {
```

Comment:
I wonder if we should reuse `BlockInfo` instead of introducing new struct. Yeah it stores extra `prev_hash`, but the readability is easier.

If separate struct is still needed, let's change the naming. `PrevBlockWithChanges` assumes that changes are inside it but it is just `BlockInfo`. Maybe `BlockWithChangesInfo`?

Reply:
Changed to use `BlockInfo`, please take a look.